containers/datalab/content/GCSbackup.sh

```diff
@@ -62,6 +62,14 @@ while [[ $# -gt 1 ]]; do
         log_file="$2"
         shift
         ;;
+      -m)
+        machine_id="$2"
+        shift
+        ;;
+      -P)
```

Comment:
Sure. Btw, I added this as for testing, as it should be detected automatically on GCE VMs. I added a comment explaining this.